layer/decorators/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .conda_decorator import conda  # noqa
```

Comment:
Can we import this in `layer/__init__.py` as well so that users can use it as `@layer.conda`?